hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -34,11 +34,12 @@
   ],
 
   alert_list: [
-    { name: "otp_macro_failure",
-      desc: "This alert triggers if hardware detects an ECC or digest error in the buffered partitions.",
-    }
-    { name: "otp_check_failure",
-      desc: "This alert triggers if the digest over the buffered registers does not match with the digest stored in OTP.",
+    { name: "fatal",
```

Comment:
Could you also change the naming for OTP_CTRL DV as well?
https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/dv/env/otp_ctrl_env_pkg.sv#L26